tests/onnx/requirements.txt

```diff
@@ -2,11 +2,9 @@ pytest-mock>=3.3.1
 pytest-cov
 pytest-dependency>=0.5.1
 pytest-ordering>=0.6
-torch~=1.13.0
-torchvision~=0.14.0
+torch==2.1.0
+torchvision==0.16.0
```

Comment:
What is the reason for these changes and the PR in general?

Reply:
To validate the latest package versions